src/main/flight/pid_init.c

```diff
@@ -39,9 +39,11 @@
 #include "flight/feedforward.h"
 #include "flight/pid.h"
 #include "flight/rpm_filter.h"
+#include "flight/imu.h"
 
 #include "sensors/gyro.h"
 #include "sensors/sensors.h"
+#include "sensors/acceleration.h"
```

Comment:
Around line 55 lower the cutoff for the attitudeFilter to 100hz

define ATTITUDE_CUTOFF_HZ 100

Reply:
done

Reply:
> Around line 55 lower the cutoff for the attitudeFilter to 100hz
> 
> define ATTITUDE_CUTOFF_HZ 100

Why? Why does it need extra filtering?